Temporarily disable Azure service tests

p8 opened the pull request:
### Summary

The Azure service tests are currently failing on the main branch, probably because of
configuration. We can temporarily disable them while we get them working again.

byroot commented on activestorage/test/controllers/blobs/redirect_controller_test.rb:
```diff
@@ -68,7 +68,7 @@ class ActiveStorage::Blobs::RedirectControllerWithOpenRedirectTest < ActionDispa
   end
 
-  if SERVICE_CONFIGURATIONS[:azure]
+  if false # SERVICE_CONFIGURATIONS[:azure]
```
Maybe add a comment as of why?

byroot replied:
Also rather than `false`, something like `!ENV['CI']`, so that if someone get their hands on the credentials they can run the suite locally.

p8 replied:
Yes, that makes sense.

p8 replied:
I've added a comment and moved the disabling to the intialization of the `SERVICE_CONFIGURATIONS` as suggested by @dhh.

p8 committed:
Temporarily disable Azure service tests

The Azure service tests are currently failing on the main branch,
probably because of configuration. We can temporarily disable them while
we get things working again.
